tools/pv_tree_generator/pop_obs_spec_common.textproto

```diff
@@ -33,7 +43,14 @@ spec: {
   obs_props: {
     mprop: "count"
     stat_type: "measuredValue"
+    name: "Count"
   }
+  #obs_props: {
+  #  mprop: "lifeExpectancy"
+  #  stat_type: "measuredValue"
+  #  name: "Life Expectancy"
+  #} # Currently not shown in the tree due to the filtered
```

Comment:
what range is filtered here? If possible, we can disable the filter for this.

Reply:
Disabled the comments. 
However, the statsVars are not showing on the tree. The reason is: we're only showing specific age ranges: eg: < 5, 5-17, 18-24. But the age ranges for lifeExpectancy are: 1, 2, 3, 4, ..; These values would be filtered when building the tree.

Reply:
We have similar treatment for different ages: https://github.com/datacommonsorg/website/blob/master/tools/pv_tree_generator/constants.py#L89-L100, would this work? Can add a new age list

Reply:
It would work if we put lifeExpectancy under a different category or at least in different specs. 
We should have two "age" nodes. Each has its own age list.

Reply:
okay, might leave a todo for this.